release_management/_utils.py

```diff
@@ -64,7 +64,7 @@ def get_package_info(manifest_abs_path, package_name):
         "metadata",
         "--format-version=1",
         "--no-deps",
-        "--manifest-path=%s" % manifest_abs_path,
+        f"--manifest-path={manifest_abs_path,}",
```

Comment:
Extra comma inside the braces.

Reply:
OK; sorry about that.  Squashed into the "Switch Python programs to literal string interpolation" commit (now that I had the chance to amend that commit, I fixed the title).